src/adapter/adapterDiscovery.ts

```diff
@@ -0,0 +1,430 @@
+ * On Windows, there are occurrences where `manufacturer` is replaced by the OS driver. Example: `ITEAD` => `wch.cn`.
+ *
+ * In virtualized environments, the passthrough mechanism can affect the `path`.
+ * Example:
+ *   Linux: /dev/serial/by-id/usb-ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20240122184111-if00
+ *   Windows host => Linux guest: /dev/serial/by-id/usb-1a86_USB_Single_Serial_54DD002111-if00
+ *
+ * XXX: vendorId `10c4` + productId `ea60` is a problem on Windows since can't match `path` and possibly can't match `manufacturer` to refine properly
+ */
+const USB_FINGERPRINTS: Record<DiscoverableUSBAdapter, USBAdapterFingerprint[]> = {
+    deconz: [
+        {
+            // Conbee II
+            vendorId: '1cf1',
+            productId: '0030',
+            manufacturer: 'dresden elektronik ingenieurtechnik GmbH',
+            // /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2132111-if00
+            pathRegex: '.*conbee.*',
+        },
+        {
+            // Conbee III
+            vendorId: '0403',
+            productId: '6015',
+            manufacturer: 'dresden elektronik ingenieurtechnik GmbH',
+            // /dev/serial/by-id/usb-dresden_elektronik_ConBee_III_DE03188111-if00-port0
+            pathRegex: '.*conbee.*',
+        },
+    ],
+    ember: [
+        // {
+        //     // TODO: Easyiot ZB-GW04 (v1.1)
+        //     vendorId: '',
+        //     productId: '',
+        //     manufacturer: '',
+        //     pathRegex: '.*.*',
+        // },
+        // {
+        //     // TODO: Easyiot ZB-GW04 (v1.2)
+        //     vendorId: '1a86',
+        //     productId: '',
+        //     manufacturer: '',
+        //     // /dev/serial/by-id/usb-1a86_USB_Serial-if00-port0
+        //     pathRegex: '.*.*',
+        // },
+        {
+            // Home Assistant SkyConnect
+            vendorId: '10c4',
+            productId: 'ea60',
+            manufacturer: 'Nabu Casa',
+            // /dev/serial/by-id/usb-Nabu_Casa_SkyConnect_v1.0_3abe54797c91ed118fc3cad13b20a111-if00-port0
+            pathRegex: '.*Nabu_Casa_SkyConnect.*',
+        },
+        // {
+        //     // TODO: Home Assistant Yellow
+        //     vendorId: '',
+        //     productId: '',
+        //     manufacturer: '',
+        //     // /dev/ttyAMA1
+        //     pathRegex: '.*.*',
+        // },
+        {
+            // SMLight slzb-07
+            vendorId: '10c4',
+            productId: 'ea60',
+            // manufacturer: '',
+            // /dev/serial/by-id/usb-Silicon_Labs_CP2102N_USB_to_UART_Bridge_Controller_a215650c853bec119a079e957a0af111-if00-port0
+            pathRegex: '.*slzb-07.*',
+        },
+        {
+            // Sonoff ZBDongle-E V2
+            vendorId: '1a86',
+            productId: '55d4',
+            manufacturer: 'ITEAD',
+            // /dev/serial/by-id/usb-ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20240122184111-if00
+            pathRegex: '.*sonoff.*plus.*',
```

Comment:
```suggestion
            pathRegex: '.*sonoff.*plus.*_v2.*',
```

To prevent it from being the same as the Dongle-P

Reply:
That was on purpose, `V2` is not always present in Dongle-E neither. (Just haven't gathered all possible combinations for the comments yet.)
But it doesn't matter since it can't conflict due to different `vendorId`+`productId`.